assets/js/common/AbilitiesMultiSelect/AbilitiesMultiSelect.jsx

```diff
@@ -82,14 +93,15 @@ function AbilitiesMultiSelect({
   userAbilities,
   placeholder,
   setAbilities,
+  operationsEnabled = getFromConfig('operationsEnabled'),
```

Comment:
I'm just putting this here as a default value. 
Otherwise, I need to pass the variable at least in other 3 components, and it is really annoying.
It is just a temporary thing anyway, until we can remove this `operationsEnabled` flag